app/views/courses/TeacherClassView.coffee

```diff
@@ -88,6 +88,7 @@ module.exports = class TeacherClassView extends RootView
 
   initialize: (options, classroomID) ->
     super(options)
+    @utils = utils
```

Comment:
What does this do?

Reply:
need the utils in template, looks like only variables in (coffee's) this can used in template (.jade/.pug)